components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -99,20 +97,21 @@ public String jwks() {
             while (enumeration.hasMoreElements()) {
                 String alias = (String) enumeration.nextElement();
                 if (keystore.isKeyEntry(alias)) {
-                    CertificateChainInfo certificateChainInfo = new CertificateChainInfo();
-                    certificateChainInfo.setCertificate(keystore.getCertificate(alias));
-                    certificateChainInfo.setCertificateChain(keystore.getCertificateChain(alias));
-                    certChainInfosWithAliases.put(alias, certificateChainInfo);
+                    CertificateInfo certificateInfo = new CertificateInfo();
+                    certificateInfo.setCertificate(keystore.getCertificate(alias));
+                    certificateInfo.setCertificateChain(keystore.getCertificateChain(alias));
+                    certificateInfo.setCertificateAlias(alias);
+                    certificateInfoList.add(certificateInfo);
                 }
             }
-            return buildResponse(certChainInfosWithAliases);
+            return buildResponse(certificateInfoList);
         } catch (Exception e) {
             String errorMessage = "Error while generating the keyset for tenant domain: " + tenantDomain;
             return logAndReturnError(errorMessage, e);
         }
     }
 
-    private String buildResponse(Map<String, CertificateChainInfo> certChainInfosWithAliases)
+    private String buildResponse(List<CertificateInfo> certInfos)
```

Comment:
```suggestion
    private String buildResponse(List<CertificateInfo> certInfos)
```

Reply:
`certInfo` is used to refer an element in the list `certInfos` later in this method.

Reply:
infos can't be used as a plural. Can we used some other variable name? `certificateInfoList`?